hadoop-ozone/dist/src/main/compose/upgrade/upgrades/non-rolling-upgrade/callbacks/common/callback.sh

```diff
@@ -35,12 +35,26 @@ validate() {
     execute_robot_test "$SCM" -v PREFIX:"$1" ${@:2} upgrade/validate.robot
 }
 
+## @description Validates that if cluster supports snapshot feature.
+## @param Whether the snapshot feature should be supported in cluster.
+## @param All parameters after the first one are passed directly to the robot command,
+##        see https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#all-command-line-options
+validate_snapshot_support() {
+  TEST_TAG="snapshot-non-support"
+  if [ "$1" = "true" ];
+  then
+    TEST_TAG="snapshot-support"
+  fi;
+  execute_robot_test --include $TEST_TAG -v ${@:2} upgrade/snapshot.robot
```

Comment:
Snapshot-related validation should be added to specific version (1.4.0), not `common`.  Upgrade test for 1.2.1 -> 1.3.0 would fail if we expected snapshots to work in the latter.

See for example EC, which was added in 1.3.0:

https://github.com/apache/ozone/blob/df4df2037143728c0b2e52844bc8f1ffbbae79f5/hadoop-ozone/dist/src/main/compose/upgrade/upgrades/non-rolling-upgrade/callbacks/1.3.0/callback.sh#L20-L30

Reply:
Yup thanks attila, I moved these validation tests into folder of 1.4.0.